lib/modules/manager/nuget/extract.ts

```diff
@@ -53,11 +53,13 @@ function extractDepsFromXml(xmlNode: XmlDocument): NugetPackageDependency[] {
 
       if (is.nonEmptyStringAndNotWhitespace(dep.depName)) {
         results.push({
+          autoReplaceStringTemplate: dep.autoReplaceStringTemplate,
           datasource: DockerDatasource.id,
           depType: 'docker',
           depName: dep.depName,
           currentValue: dep.currentValue,
           currentDigest: dep.currentDigest,
+          replaceString: dep.replaceString,
         });
```

Comment:
```suggestion
        results.push(dep);
```

why not simply do this?